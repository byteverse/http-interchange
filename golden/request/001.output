PUT /foo/bar?baz=true HTTP/1.1

