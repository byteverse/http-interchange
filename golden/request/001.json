{
    "requestLine": {
        "method": "PUT",
        "path": "/foo/bar?baz=true",
        "versionMajor": 1,
        "versionMinor": 1
    },
    "headers": []
}
